openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-cmdtlmserver/src/tools/CmdTlmServer/RawDialog.vue

```diff
@@ -23,64 +23,45 @@
 <template>
   <div
     v-show="isVisible"
+    ref="rawDialog"
     :style="computedStyle"
     class="raw-dialog"
-    ref="rawDialog"
   >
     <div ref="bar" class="toolbar-wrapper">
       <v-toolbar height="24">
-        <v-tooltip location="top">
-          <template v-slot:activator="{ props }">
-            <div v-bind="props">
-              <v-icon data-test="copy-icon" @click="copyRawData">
-                mdi-content-copy
-              </v-icon>
-            </div>
-          </template>
-          <span> Copy </span>
-        </v-tooltip>
-        <v-tooltip location="top">
-          <template v-slot:activator="{ props }">
-            <div v-bind="props">
-              <v-icon data-test="download" @click="downloadRawData">
-                mdi-download
-              </v-icon>
-            </div>
-          </template>
-          <span> Download </span>
-        </v-tooltip>
+        <v-btn
+          icon="mdi-content-copy"
+          variant="text"
+          data-test="copy-icon"
+          @click="copyRawData"
+        />
```

Comment:
Ok so instead of a `v-icon` with `@click` we're using an actual `v-btn` with the `icon` prop. That's more correct since we're using these as buttons.